.github/workflows/compatibility-updater.yaml

```diff
@@ -0,0 +1,61 @@
+name: Utility / Compatibility Matrix Update
+
+on:
+  schedule:
+    # Runs at 00:00 UTC every day
+    - cron: '0 0 * * *'
+
+jobs:
+  run-compatibility-updater:
+    runs-on: ubuntu-latest
+
+    env:
+      SCRIPTS_DIR: utils/compatibility
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Install dependencies
+      run: |
+        # Install pyenv dependencies
+        sudo apt-get update
+        sudo apt-get install -y make build-essential libssl-dev zlib1g-dev \
+            libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm \
+            libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev \
+            liblzma-dev
+
+    - name: Install pyenv
+      run: |
+        curl https://pyenv.run | bash
+    
+    - name: Set up Env and Run script
+      run: |
+        export PYENV_ROOT="$HOME/.pyenv"
+        export PATH="$PYENV_ROOT/bin:$PATH"
+        eval "$(pyenv init --path)"
+        eval "$(pyenv init -)"
+        cd $SCRIPTS_DIR
+        source setup_env.sh
+        python main.py    
+
+    - name: Get current date
+      id: date
+      run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+
+    - name: Commit Changes
```

Comment:
the create-pull-request action should commit for you

Reply:
I removed the `Commit Changes` step and tested, it's working as expected